src/features/FindReferencesManager.js

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright (c) 2019 - present Adobe. All rights reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+ * FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+ * DEALINGS IN THE SOFTWARE.
+ *
+ */
+
+define(function (require, exports, module) {
+    "use strict";
+
+    var AppInit                     = require("utils/AppInit"),
+        CommandManager              = require("command/CommandManager"),
+        EditorManager               = require("editor/EditorManager"),
+        Menus                       = require("command/Menus"),
+        ProviderRegistrationHandler = require("features/PriorityBasedRegistration").RegistrationHandler,
+        SearchResultsView           = require("search/SearchResultsView").SearchResultsView,
+        SearchModel                 = require("search/SearchModel").SearchModel,
+        Strings                     = require("strings");
+
+    var _providerRegistrationHandler = new ProviderRegistrationHandler(),
+        registerFindReferencesProvider = _providerRegistrationHandler.registerProvider.bind(
+            _providerRegistrationHandler
+        ),
+        removeFindReferencesProvider = _providerRegistrationHandler.removeProvider.bind(_providerRegistrationHandler);
+
+    var SHOW_FIND_REFERENCES_CMD_ID   = "showReferences",
```

Comment:
Since this is a core feature, move this to [Commands.js](https://github.com/adobe/brackets/blob/master/src/command/Commands.js#L90)

Change string to `"cmd.findAllReferences"`. This will allow you to use it in DefaultMenus.js.